client/components/public/Home.jsx

```diff
@@ -25,13 +25,14 @@ const CoronavirusMessage = (
                     </Message.Header>
                     <Segment basic size='large' className='no-padding' style={{lineHeight: "normal"}}>
                     <ul>
-                      <li>The FIFTH Annual WWU Great Puzzle Hunt will be held VIRTUALLY</li>
+                      <li>The FIFTH Annual WWU Great Puzzle Hunt will be held <b>VIRTUALLY</b></li>
                       <li>On Saturday, April 17, 2021</li>
-                      <li>Registration will open in January</li>
-                      <li>The event will be FREE and open to all</li>
-                      <li>Information will be updated this fall</li>
+                      <li>Registration and Gear Sales open January 5, 2021</li>
```

Comment:
don't we have a `registrationOpenDate` or something similar in the global variables?

Reply:
ah, here it is: `regularRegistrationStart`

Reply:
Now using the eventDate and regularRegistrationStart meteor variables